src/providers/aws.rs

```
@@ -556,6 +556,8 @@ where
 
                 futures_util::future::join_all(self.regions.drain().map(|(region, mut rl)| {
                     let region_span = tracing::debug_span!("region", %region);
+                    // this throws away a possible error from terminate_all,
+                    // but that is ok.
```

Why is it okay? Wouldn't it be better to propagate that error with `?`?